app/assets/javascripts/osem-datatables.js

```diff
@@ -1,6 +1,6 @@
 $(function () {
   $(document).ready(function() {
-    $('.datatable').DataTable({
+    $('.data-table').DataTable({
```

Comment:
There is  a solution for handling that case with datatables described in the link provided by the error.